src/rebase.c

```diff
@@ -1327,6 +1327,33 @@ int git_rebase_finish(
 	return error;
 }
 
+static int git_rebase_get_annotated_name(git_annotated_commit **out, const char *name, git_rebase *rebase) {
+	int error;
+	git_reference *ref;
+
+	if (rebase->type != GIT_REBASE_TYPE_MERGE) {
+		git_error_set(GIT_ERROR_REBASE, "invalid rebase type");
+		return -1;
+	}
+
+	if ((error = git_reference_dwim(&ref, rebase->repo, name)))
```

Comment:
Hmm, I don't like that an external change could cause a failure there… But we'd be in a lot of trouble already if that was the case.